src/utils/setCommentForPathtest.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from "chai";
+import setCommentForPath from "./setCommentForPath";
+import { FileType } from "../tree/types";
+
+let treeCore = [
+  {
+    text: "src/utils",
+    depthLevel: 2,
+    fileType: FileType.FOLDER,
+    comment: " ",
+  },
+  {
+    text: "src",
+    depthLevel: 1,
+    fileType: FileType.FOLDER,
+    comment: " # Source files",
+  },
+  {
+    text: "src/components",
+    depthLevel: 2,
+    fileType: FileType.FOLDER,
+    comment: "ss ",
+  },
+];
+
+describe("setCommentForPath Function", () => {
+  it("full address", () => {
+    setCommentForPath(treeCore, "src", "NEW COMMENT!");
+    expect(treeCore).to.equal([
+      {
+        text: "src/utils",
+        depthLevel: 2,
+        fileType: FileType.FOLDER,
+        comment: " ",
```

Comment:
Where is the treeCore of the folder?
Also fileType is not stored in the tree, it is returned by a function that takes in the text :) (PATH)

Reply:
I haven't change the type.ts

Reply:
@ralph-dev so since you said PRs shouldn't cause any conflict, I haven't made change in type.ts 
should we change it?
we should I think @Manpreet-Bhatti should be aware too to make sure he writes his codes accordingly

Reply:
@ralph-dev another problem is that it missing "deletedOrde"r as well and it got "depthLevel"

Reply:
Yeah but you can just look at the architecture (NEW ONE) that doesn't have either of those fields :)